util/src/main/i18n/templates/command.properties

```diff
@@ -77,4 +77,9 @@ command.list.participants = Participants
 command.message.noReply = Did not find a message to reply to, use {0}
 
 # {0} = player name
-command.message.blocked = {0} is not accepting messages at this time.
\ No newline at end of file
+command.message.blocked = {0} is not accepting messages at this time.
+
+# {0} = # number of map pools found
+command.admin.reloadPools = Successfully reloaded {0}
```

Comment:
`Successfully reloaded {0} pools` ?

Reply:
I actually had it so `mapPools.title` & `mapPools.name` were appended to the end in order to translate the words. But I’m removing it anyway 🤷‍♂️